sys-libs/glibc/README.md

```diff
@@ -1,9 +0,0 @@
-# GLibc
```

Comment:
`sys-libs/glibc/README.md` has disappeared, and not added again later.
Is that intended?

Reply:
Hm, no. Something must went wrong. Will fix it.

Reply:
It's back.